codespell_lib/data/dictionary.txt

```diff
@@ -11931,6 +11965,9 @@ paticular->particular
 paticularly->particularly
 patrent->parent, patent, patron,
 pattented->patented
+patter->pattern
```

Comment:
This is a word, but maybe we should keep since there will presumably be way more true positives than false positives from including this

Reply:
It feels like a slippery slope to me; if you're going to do that, can they at least be pulled into a separate dictionary file instead and add the ability to use multiple dictionaries (if not already present). Then there's just the challenge of deciding what typos are sufficiently popular to be in the second file.

Reply:
What should I do? 
Delete the word?

Reply:
Yeah

Reply:
Okay. 
'patter' & 'patters' deleted.